cypress/e2e/admin/adminRegistersTeacherWithLink.feature

```diff
@@ -1,37 +1,69 @@
 @regression_test
-@unstable_test
+@stable_test
 Feature: Teacher registration with registration link send by admin
```

Comment:
same comments from the student registeration feature

Reply:
done!